docs/checks.md

```diff
@@ -2,52 +2,68 @@
 ```
 
-> **Note**: `Command*` checks are prone to interception, modification, and tomfoolery. Your scoring configuration will be much more robust if you rely on checks using native mechanisms rather than shell commands (for example, `PathExists` instead of ls).
-> **Note**: If any check returns an error (e.g., something that it was not expecting), it will _never_ pass, even if it's a `Not` condition. This varies by check, but for example, if you try to check the content of a file that doesn't exist, it will return an error and not succeed-- even if you were doing `FileContainsNot`.
+> **Note**: `Command*` checks are prone to interception, modification, and tomfoolery. Your scoring configuration will
+> be much more robust if you rely on checks using native mechanisms rather than shell commands (for
+> example, `PathExists`
+> instead of ls).
 
-**CommandOutput**: pass if command output matches string exactly. If it returns an error, check never passes. Use of this check is discouraged.
+> **Note**: If any check returns an error (e.g., something that it was not expecting), it will _never_ pass, even if
+> it's a `Not` condition. This varies by check, but for example, if you try to check the content of a file that doesn't
+> exist, it will return an error and not succeed-- even if you were doing `FileContainsNot`.
+
+**CommandOutput**: pass if command output matches string exactly. If it returns an error, check never passes. Use of
+this check is discouraged.
 
 ```
 type = 'CommandOutput'
 cmd = '(Get-NetFirewallProfile -Name Domain).Enabled'
 value = 'True'
 ```
 
-**DirContains**: pass if directory contains regular expression (regex) string
-
-> **Note**: Read more about regex [here](regex.md).
+**DirContains**: pass if directory contains a string value
 
 ```
 type = 'DirContains'
 path = '/etc/sudoers.d/'
 value = 'NOPASSWD'
 ```
 
-> `DirContains` is recursive! This means it checks every folder and subfolder. It currently is capped at 10,000 files, so you should begin your search at the deepest folder possible.
+> `DirContains` is recursive! This means it checks every folder and subfolder. It currently is capped at 10,000 files,
+> so you should begin your search at the deepest folder possible.
 
 
-> **Note**: You don't have to escape any characters because we're using single quotes, which are literal strings in TOML. If you need use single quotes, use a TOML multi-line string literal `''' like this! that's neat! C:\path\here '''`), or just normal quotes (but you'll have to escape characters with those).
+> **Note**: You don't have to escape any characters because we're using single quotes, which are literal strings in
+> TOML. If you need use single quotes, use a TOML multi-line string
+> literal `''' like this! that's neat! C:\path\here '''`), or just normal quotes (but you'll have to escape characters
+> with those).
 
-**FileContains**: pass if file contains regex
+**FileContains**: pass if file contains a value
```

Comment:
We still need documentation for FileContainsRegex and all the other *Regex checks right? Since I dont think you can 100% guess if all checks have a regex alternate

Reply:
![image](https://github.com/elysium-suite/aeacus/assets/45888585/db178e9c-1baf-459f-a564-e34add3bb10d)

Reply:
![image](https://github.com/elysium-suite/aeacus/assets/45888585/48d51069-4bd8-47e1-b9cd-e9fe64aa57c5)

Reply:
Fair point thanks I definitely skimmed over the note block we should probably make that more readable at some point